lib/services/network/authorization/identity_bloc.dart

```diff
@@ -7,32 +6,35 @@ import 'package:rxdart/rxdart.dart';
 
 class IdentityBloc {
   final _identitySubject = PublishSubject<User>();
-  Stream <User> get identityStream => _identitySubject.stream;
+  Stream<User> get identityStream => _identitySubject.stream;
 
-  identify() async{
+  identify() async {
     User res = await repo.getIdentity();
     logger.d("response for login: ${res}");
-    switch (res.statusCode){
-      case 200: {
-        _identitySubject.sink.add(res);
-        // TODO - sink success.
-      }
-      break;
-      case 401: {
-        _identitySubject.sink.addError(NetworkError(res.statusCode, "One of the credentials is incorrect!"));
-      }
-      break;
-      default: {
-        _identitySubject.sink.addError(NetworkError(res.statusCode, "An unknown error has occurred."));
-        //TODO - Sink error
-      }
+    switch (res.statusCode) {
+      case 200:
+        {
+          _identitySubject.sink.add(res);
+          // TODO - sink success.
+        }
+        break;
+      case 401:
+        {
+          _identitySubject.sink.addError(NetworkError(
+              res.statusCode, "One of the credentials is incorrect!"));
```

Comment:
Consider using constants. Should be a file /app/src/main/res/values/strings.xml that defines them